Watch Extension/AppState.swift

```diff
@@ -8,6 +8,25 @@ struct Feeding: Identifiable {
     let start: Date
     let type: FeedingType
     let side: FeedingSide
+    var lastPausedDate: Date? = nil
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/880c027cb3a63d59b8015ec9e1cab691a35d0496/Watch Extension/AppState.swift#L11"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>AppState.swift:11</code></td>
    </tr>
  </tbody>
</table>